react/package.json

```diff
@@ -26,6 +24,12 @@
     "@types/node": "^12.6.2",
     "@types/react": "^16.8.23",
     "@types/react-dom": "^16.9.0",
-    "@types/recharts": "^1.8.0"
+    "@types/recharts": "^1.8.0",
+    "recharts": "^2.0.0-beta.1"
```

Comment:
Also, do you have some specific reason to use this beta version of recharts?

Reply:
And should this be in both `devDependencies` and `peerDependencies`?

Reply:
> Also, do you have some specific reason to use this beta version of recharts?

The beta version was already being used, the Fanny used it, so I didn't want to change the version, just put it as peerDependencies.

> And should this be in both devDependencies and peerDependencies?

I explained in my last comment https://github.com/vtex/styleguide/pull/1048#issuecomment-587179160

Reply:
the ideal scenario would be for our documentation to be separate from our styleguide.